lib/validation.js

```diff
@@ -76,32 +76,42 @@ function validateParam (validatorFunction, request, paramName) {
 
 function validate (context, request) {
   var params = validateParam(context[paramsSchema], request, 'params')
+  var schemaErrorFormatter = context.schemaErrorFormatter || schemaErrorsText
+
   if (params) {
-    return wrapValidationError(params, 'params')
+    return wrapValidationError(params, 'params', schemaErrorFormatter)
```

Comment:
I would bind the `schemaErrorFormatter`
so the user may access the request with `this`

Because the dev doesn't know in which route the error has happened (it could be useful)